packages/embeddable/webpack.config.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const BundleAnalyzerPlugin = require("webpack-bundle-analyzer")
+  .BundleAnalyzerPlugin;
+const path = require("path");
+
+module.exports = {
+  mode: "development",
+  entry: path.resolve(__dirname, "./src/index.tsx"),
+  output: {
+    path: path.resolve(__dirname, "./dist"),
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: "src/index.html",
+    }),
+    process.env.ANALYZE && new BundleAnalyzerPlugin(),
+  ].filter(Boolean),
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        exclude: /(node_modules|bower_components)/,
```

Comment:
Do we also need to exclude tests folder?

Reply:
probably